tests/unit/test_table.py

```diff
@@ -3534,7 +3537,10 @@ def test_to_dataframe_no_tqdm(self):
         user_warnings = [
             warning for warning in warned if warning.category is UserWarning
         ]
-        self.assertEqual(len(user_warnings), 1)
+        # Note: number of warnings is inconsistent across python versions
```

Comment:
I removed the assertion for number of warnings, because we get different number of them with different python versions.